### PR TITLE
Update set-vmnetworkadaptervlan.md

### PR DESCRIPTION
**Pull request by dariomws306**
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/560

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/hyper-v/set-vmnetworkadaptervlan.md [hunk 1]
```diff
@@ -350,7 +357,7 @@ Accept wildcard characters: False
 ```
 
 ### -Trunk
-Specifies **Trunk** mode for the virtual machine network adapter.
+Specifies **Trunk** mode for the virtual machine network adapter. This parameter configures a tagged virtual port that passes all allowed VLANId tags to the VM adapter, traffic with the NativeVLANId is passed untagged to the VM adapter.
```
Comment (nenonix) [inline docset/windows/hyper-v/set-vmnetworkadaptervlan.md]:
```suggestion
Specifies **Trunk** mode for the virtual machine network adapter. This parameter configures a tagged virtual port that passes all allowed VLANId tags to the VM adapter. Traffic with the NativeVLANId is passed untagged to the VM adapter.
```

Comment (JohanFreelancer9) [reply docset/windows/hyper-v/set-vmnetworkadaptervlan.md]: agree

**Comment by e0i** [comment]
@tseward @get-itips Peer review needed for this PR. Thanks.

**Comment by get-itips** [comment]
I suggest to change "This parameter configures an untagged virtual port for the VLANId" with "...untagged virtual port with the VLANId specified"

**Comment by e0i** [comment]
@tseward Any thoughts?

**Comment by e0i** [comment]
@dariomws306 Please follow-up with Andres's suggestions. Thanks.

**Comment by dariomws306** [comment]
@e0i, done!

**Comment by get-itips** [comment]
Ok!

File: docset/windows/hyper-v/set-vmnetworkadaptervlan.md [hunk 2]
```diff
@@ -103,11 +103,18 @@ PS C:\> Get-VM Redmond | Set-VMNetworkAdapterVlan -Untagged
...
+
+Sets the virtual switch in the management OS to the Access mode.
+Traffic sent by this virtual switch is tagged with VLAN ID 20.
+
 ## PARAMETERS
 
 ### -Access
-Specifies **Access** mode for the virtual machine network adapter.
-This parameter must be specified in conjunction with parameter **VlanId**.
+Specifies **Access** mode for the virtual machine network adapter. This parameter configures an untagged virtual port with the VLANId (port based VLAN), for this reason must be specified in conjunction with parameter **VlanId**.
```
Comment (nenonix) [inline docset/windows/hyper-v/set-vmnetworkadaptervlan.md]:
```suggestion
Specifies **Access** mode for the virtual machine network adapter. This parameter configures an untagged virtual port with the VLANId (port based VLAN), so it must be specified together with **VlanId**.
```

Comment (dariomws306) [reply docset/windows/hyper-v/set-vmnetworkadaptervlan.md]: I am not sure about this... Maybe:

"..., for this reason must be specified together with the **VlanId** parameter."

What do you think @get-itips @tseward @nenonix?

Comment (get-itips) [reply docset/windows/hyper-v/set-vmnetworkadaptervlan.md]: Both are ok I think

**Comment by e0i** [comment]
@tseward Any updates?

**Comment by tseward** [comment]
I'm good with the changes.

**Comment by e0i** [comment]
@AndreaBarr Please merge this PR that has passed peer review. Thanks.

**Pull request merged**